docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
     build:
       context: ./build/deps
       dockerfile: mongo6.Dockerfile
-    command: --setParameter enableTestCommands=1
+    command: --config /etc/mongod.conf --setParameter enableTestCommands=1
```

Comment:
While we are on it, could we move `enableTestCommands=1` to the config file too?